prog/dftb+/lib_type/typegeometry.F90

```diff
@@ -8,10 +8,12 @@
 !> data type and associated routines for specifying atomic geometry and boundary conditions
 module typegeometry
   use accuracy
+  use lapackroutines
   implicit none
   private
 
   public :: TGeometry, normalize
+  public :: reduce, setLattice
```

Comment:
Can reduce have a more informative name?

Reply:
It reduces the full geometry to a subset (e.g. contact). :wink:  If you have any better suggestion, feel free to rename.